src/components/EmojiPicker/index.js

```diff
@@ -33,6 +33,14 @@ class EmojiPicker extends React.Component {
         this.emojiPopoverDimensionListener = Dimensions.addEventListener('change', this.measureEmojiPopoverAnchorPosition);
     }
 
+    // eslint-disable-next-line rulesdir/prefer-early-return
+    componentDidUpdate(prevProps, prevState) {
+        /** Dismiss the Keyboard and focus to prevent emoji press twice when selecting an emoji */
```

Comment:
Comment should be in the form of `//`. The comment style you used is only for propTypes.